Example.iOS/Security/SecurityCheckProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using AeroGear.Mobile.Security;
+using Example.Security;
+
+[assembly: Xamarin.Forms.Dependency(typeof(Example.iOS.Security.SecurityCheckProvider))]
+namespace Example.iOS.Security
+{
+    public class SecurityCheckProvider : ISecurityCheckProvider
+    {
+        public SecurityCheckProvider()
+        {
+        }
+
+        public IList<ISecurityCheckType> SecurityChecks => new List<ISecurityCheckType> {
+            AeroGear.Mobile.Security.SecurityChecks.NOT_JAILBROKEN,
+            AeroGear.Mobile.Security.SecurityChecks.DEVICE_LOCK,
+            AeroGear.Mobile.Security.SecurityChecks.NOT_IN_EMULATOR,
+            AeroGear.Mobile.Security.SecurityChecks.NO_DEBUGGER
```

Comment:
You can use [this](https://github.com/aerogear/aerogear-xamarin-sdk/blob/master/Security/Security.Platform.iOS/SecurityChecks.cs#L70) instead of enlisting manually